src/layouts/Footer.js

```diff
@@ -17,47 +18,43 @@ const useStyles = makeStyles((theme) => ({
     fontWeight: "bold",
     textAlign: "center",
     textTransform: "capitalize",
-  },
-  footerlink: {
-    "&:hover": {
+    "&:hover *": {
       textDecoration: "none",
-    }
-  }
+    },
+  },
 }));
 
-function Footer() {
-  const classes = useStyles();
-  const preventDefault = (event) => event.preventDefault();
-  const links = [
-    {
-      text: "Copyright",
-      url: "#",
-      id: 1
-    },
-    {
-      text: "Ochrona danych osobowych",
-      url: "#",
-      id: 2
-    },
-    {
-      text: "Polityka prywatności",
-      url: "#",
-      id: 3
-    },
+const links = [
+  {
+    text: "Ochrona danych osobowych",
+    url: "/personal_data_protection",
```

Comment:
when you make const with routes, you could use variable here like: `    url: "paths.pageX",`

Reply:
Added routing.js and imported paths.